profiler/src/main/java/com/splunk/opentelemetry/profiler/exporter/PprofCpuEventExporter.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.profiler.exporter;
+
+import static com.splunk.opentelemetry.profiler.ProfilingSemanticAttributes.SOURCE_EVENT_NAME;
+import static com.splunk.opentelemetry.profiler.ProfilingSemanticAttributes.SOURCE_EVENT_PERIOD;
+import static com.splunk.opentelemetry.profiler.ProfilingSemanticAttributes.SOURCE_EVENT_TIME;
+import static com.splunk.opentelemetry.profiler.ProfilingSemanticAttributes.SPAN_ID;
+import static com.splunk.opentelemetry.profiler.ProfilingSemanticAttributes.THREAD_ID;
+import static com.splunk.opentelemetry.profiler.ProfilingSemanticAttributes.THREAD_NAME;
+import static com.splunk.opentelemetry.profiler.ProfilingSemanticAttributes.THREAD_NATIVE_ID;
+import static com.splunk.opentelemetry.profiler.ProfilingSemanticAttributes.THREAD_STATE;
+import static com.splunk.opentelemetry.profiler.ProfilingSemanticAttributes.TRACE_ID;
+
+import com.google.perftools.profiles.ProfileProto.Sample;
+import com.splunk.opentelemetry.profiler.Configuration.DataFormat;
+import com.splunk.opentelemetry.profiler.ProfilingDataType;
+import com.splunk.opentelemetry.profiler.context.StackToSpanLinkage;
+import com.splunk.opentelemetry.profiler.events.EventPeriods;
+import com.splunk.opentelemetry.profiler.exporter.StackTraceParser.StackTrace;
+import com.splunk.opentelemetry.profiler.pprof.Pprof;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.sdk.logs.LogProcessor;
+import io.opentelemetry.sdk.resources.Resource;
+import java.time.Duration;
+import java.time.Instant;
+
+public class PprofCpuEventExporter implements CpuEventExporter {
+  private final DataFormat dataFormat;
+  private final EventPeriods eventPeriods;
+  private final PprofLogDataExporter pprofLogDataExporter;
+  private Pprof pprof = createPprof();
+
+  private PprofCpuEventExporter(Builder builder) {
+    this.dataFormat = builder.dataFormat;
+    this.eventPeriods = builder.eventPeriods;
+    this.pprofLogDataExporter =
+        new PprofLogDataExporter(
+            builder.logProcessor, builder.resource, ProfilingDataType.CPU, builder.dataFormat);
```

Comment:
This is because initially we had more formats. I'd keep it in case we need to experiment with different format, it would be nice if we could get rid of the base64.